src/main/java/wooteco/subway/exception/SubwayException.java

```diff
@@ -0,0 +1,8 @@
+package wooteco.subway.exception;
+
+public class SubwayException extends RuntimeException {
```

Comment:
```suggestion
public abstract class SubwayException extends RuntimeException {
```

SubwayException은 상속으로만 사용하고 있는데, 계층을 위해서만 사용한다면 abstract를 통해 의도를 전달할 수 있습니다 :) 
추가적으로 저는 모든 클래스를 abstract 또는 final로만 작성하는데 그 이유에 대해 고민해보면 좋을 것 같아요 ㅎㅎ